From 925d090baf17a0600b2485cec96453c24fa3f16a Mon Sep 17 00:00:00 2001
In-Reply-To: <xmqqtut6qf7q.fsf@gitster.c.googlers.com>
References: <xmqqtut6qf7q.fsf@gitster.c.googlers.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 1 Dec 2020 10:32:27 +0100
Subject: [PATCH] cook: add github.com/git-vcs/git as a URL
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

I maintain this mirror, but hopefully we can make it semi-official. It
has the same refs as the GitHub one except for the GitHub "pull" refs:

    $ diff -u \
        <(git ls-remote https://gitlab.com/git-vcs/git.git/) \
        <(git ls-remote https://github.com/git/git/|grep -v refs/pull)
    $

Although I had to delete the stale "pu" manually just now.

As an aside there are also https://gitlab.com/gitlab-org/git and
https://gitlab.com/git-vcs/git which mostly mirror but also carry some
GitLab Employee topic branches.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 MaintNotes | 1 +
 cook       | 5 +++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/MaintNotes b/MaintNotes
index 0dc03080de..bb3064e9ac 100644
--- a/MaintNotes
+++ b/MaintNotes
@@ -136,6 +136,7 @@ My public git.git repositories are (mirrored) at:
   https://kernel.googlesource.com/pub/scm/git/git
   git://repo.or.cz/alt-git.git/
   https://github.com/git/git/
+  https://gitlab.com/git-vcs/git/
 
 This one shows not just the main integration branches, but also
 individual topics broken out:
diff --git a/cook b/cook
index 03ac0cfbe4..2258390114 100755
--- a/cook
+++ b/cook
@@ -295,8 +295,8 @@ the integration branches, but I am still holding onto them.
 
 
 Copies of the source code to Git live in many repositories, and the
-following is a list of the ones I push into.  Some repositories have
-only a subset of branches.
+following is a list of the ones I push into or their mirrors.  Some
+repositories have only a subset of branches.
 
 With maint, master, next, seen, todo:
 
@@ -304,6 +304,7 @@ With maint, master, next, seen, todo:
 	git://repo.or.cz/alt-git.git/
 	https://kernel.googlesource.com/pub/scm/git/git/
 	https://github.com/git/git/
+	https://gitlab.com/git-vcs/git/
 
 With all the integration branches and topics broken out:
 
-- 
2.29.2.222.g5d2a92d10f8

